src/main/java/peoplesoft/model/money/Rate.java

```diff
@@ -37,8 +37,10 @@
 @JsonSerialize(using = Rate.RateSerializer.class)
 @JsonDeserialize(using = Rate.RateDeserializer.class)
 public class Rate {
-    public static final String MESSAGE_CONSTRAINTS = "Value for rate should be a non-negative decimal number.";
-    public static final String MESSAGE_TOO_LARGE = "Value for rate is too large.";
+    public static final String MESSAGE_CONSTRAINTS = "The rate should not be negative. "
+            + "It can include 2 decimal places.";
```

Comment:
maybe something like "The rate should not be negative or have more than 2 decimal places"?